Controller/LogsAdminController.php

```diff
@@ -39,26 +40,21 @@ public function __construct(EncryptorInterface $encryptor)
     }
 
     /**
-     * {@inheritdoc}
+     * @param Request $request
+     *
+     * @return Response
      */
-    public function decryptAction(Request $request)
+    public function decryptAction(Request $request): Response
```

Comment:
`__invoke`?

Reply:
not sure about that because:
- AdminControllers may contain several actions
- internal behavior of sonata (for now) will look for a <name>Action based on route name definition in admin class so __invoke will not work

Reply:
So `submitAction`?

Reply:
i set it to decryptEncryptAction as it is what it does